Mathlib/RepresentationTheory/GroupCohomology/LowDegree.lean

```diff
@@ -315,13 +320,23 @@ def twoCoboundaries : Submodule k (twoCocycles A) :=
 
 variable {A}
 
-theorem mem_oneCoboundaries_of_dZero_apply (x : A) :
-    ⟨dZero A x, LinearMap.ext_iff.1 (dOne_comp_dZero A) x⟩ ∈ oneCoboundaries A :=
-  LinearMap.mem_range_self _ _
+/-- Makes a 1-coboundary out of `f ∈ Im(d⁰)`. -/
+def oneCoboundariesOfMemRange (f : G → A) (h : f ∈ LinearMap.range (dZero A)) :
+    oneCoboundaries A :=
+  ⟨⟨f, LinearMap.range_le_ker_iff.2 (dOne_comp_dZero A) h⟩,
+    by rcases h with ⟨x, rfl⟩; exact ⟨x, rfl⟩⟩
 
-theorem mem_oneCoboundaries_of_mem_range (f : G → A) (h : f ∈ LinearMap.range (dZero A)) :
-    ⟨f, LinearMap.range_le_ker_iff.2 (dOne_comp_dZero A) h⟩ ∈ oneCoboundaries A := by
-  rcases h with ⟨x, rfl⟩; exact ⟨x, rfl⟩
+theorem oneCoboundaries_of_mem_range_apply (f : G → A) {h : f ∈ LinearMap.range (dZero A)} :
+    (oneCoboundariesOfMemRange f h).1.1 = f := rfl
+
+/-- Makes a 1-coboundary out of `f : G → A` and `x` such that
+`ρ(g)(x) - x = f(g)` for all `g : G`. -/
+def oneCoboundariesOfEq (f : G → A) (x : A) (hf : ∀ g, A.ρ g x - x = f g) :
```

Comment:
`f` can be implicit, but as you wish.

Reply:
I know `f`'s in the type of `hf`, but I feel like I've been annoyed in the past by things with implicit data arguments where I then have to start the proof of some hypothesis with `show...`. I can't remember the context though so it may be irrelevant, and I guess in Lean 4 it's easier to just specify `f` with `(f := ...)` if you want to.

Reply:
If you find this version more convenient just keep it like that.